Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.Properties.cs

```diff
@@ -111,6 +116,15 @@ public double CompactPaneLength
         /// <summary>
         /// Gets or sets the Brush to apply to the background of the Pane area of the control.
         /// </summary>
+        public Brush AccentColor
```

Comment:
A couple more examples, in 'Groove Music' and 'Movies & TV' the app does not have an AccentColor. Just a PaneBackground. The Back Button defaults to the system theme color.